### ISSUE TITLE
Add: CI commands need to run everything headless

### ISSUE THREAD

**Issue by digitaldrummerj** [briebug/cypress-schematic#6]
In order to run headless you need to start up a web server, wait for Angular to be available, run the Cypress tests and then stop the web server.  

See https://medium.com/ngconf/angular-adding-cypress-ui-tests-to-your-devops-pipeline-c7613e67d37d for all of the work that needs to be done to enable CI.

### dev dependencies

```json
 "start-server-and-test": "^1.7.11",
 "npm-run-all": "^4.1.5",
"http-server": "^0.11.1",
```

### package.json scripts 

```json
  "scripts": {
    "build:prod": "ng build --prod",
    "test": "ng test",
    "lint": "ng lint",
    "lint:ci": "ng lint",
    "ci": "run-s lint:ci build:prod ci:cy-run",
    "ci:cy-run": "start-server-and-test ci:start-server http://localhost:4200 cy:run",
    "ci:start-server": "http-server ./dist/CypressCi -a localhost -p 4200 -c-1",
    "cy:open": "cypress open",
    "cy:run": "cypress run",
    "cy:verify": "cypress verify"
  },
```

**Comment by anthonymjones** [comment]
Thanks, Justin. Do you have any interest in opening a PR for this and #5?

**Comment by digitaldrummerj** [comment]
@anthonymjones sure.  I can make an attempt at it.  This would make a good topic to stream on one of my [Twitch streams](https://twitch.tv/digitaldrummerj)

**Comment by anthonymjones** [comment]
Closing this issue... I just released v3.0.0. Please let me know if this is still an issue and should be re-opened. Thanks!
